backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/SparkPlanExecApiImpl.scala

```diff
@@ -340,24 +354,19 @@ class SparkPlanExecApiImpl extends SparkPlanExecApi {
       child: SparkPlan,
       numOutputRows: SQLMetric,
       dataSize: SQLMetric): BuildSideRelation = {
-    mode match {
-      case IdentityBroadcastMode =>
-        throw new IllegalStateException("Unreachable code")
```

Comment:
Have we removed this case match as it was unreachable?

Reply:
Yes, the flow is same for both modes.